libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3105,7 +3105,7 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     type: "CryptoCurrency",
     id: "ethereum_goerli",
     coinType: 60,
-    name: "Ethereum Goerli",
+    name: "Goerli",
```

Comment:
we are sure about this?

Reply:
Good question, but I changed it because the name displayed on the Nano is actually `Goerli`. So there's currently a mismatch at signature step with LL that displays "Ethereum Goerli". I was wondering about other side effects of changing the name, but seems safe no?

Reply:
I reverted that change, I guess we can live with it for testnets...